app/components/github-repo.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+import recordsList from 'code-corps-ember/utils/records-list';
+
+const {
+  computed,
+  Component,
+  get
+} = Ember;
+
+export default Component.extend({
+  classNames: ['github-repo'],
+  tagName: 'li',
+
+  isConnected: computed('githubRepo', 'project.projectGithubRepos.@each', function() {
+    let githubRepo = get(this, 'githubRepo');
+    let project = get(this, 'project');
+    let projectGithubRepos = get(this, 'project.projectGithubRepos');
+    if (projectGithubRepos) {
+      return recordsList.includes(projectGithubRepos, githubRepo);
+    }
```

Comment:
Generally, I'm not sure I love the `recordsList` utility class. It feels like we made a class just to reduce the line count, at a great cost to clarity. If there's some code repetition somewhere, that's fine. We don't have to generalize everything.

I'd write it here first, then, if it turns out the code is identical to some other piece of code, we generalize.